index.bs

```diff
@@ -117,7 +119,7 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
 <ul>
 <li>[=Sequence Header OBUs=] SHOULD not be present in the [=AV1 Item Configuration Property=].</li>
-<li>The values of the fields in the [=AV1CodecConfigurationBox=] SHALL match those of the [=Sequence Header OBU=] in the item data and in the [=AV1CodecConfigurationBox=], if any.</li>
```

Comment:
There was some grammar error in the sentence, and I couldn't make sense of it, so I cleaned it up. The original sentence basically said that that "the fields in A shall match those of B and in A". Of course A matches A, so that is redundant and was removed.

Reply:
I think I understand now what the original sentence was trying to say. I have added a separate bullet item to make it clearer.

Reply:
Sorry the last A was meant to be configOBUs. We may have the same information at 3 places: 
- in the Configuration record as direct fields
- in the OBUs in the configOBUs field of the Configuration record
- in the OBUs in the item data
They all shall match.

Reply:
Agreed, and I believe the update I made last week already covers what you say. But I think you want to distinguish between the "image item" and the data associated with that item. I think that is the only real difference between what I wrote and what you are asking for.
Since "image data" is not a term that defined anywhere, I am defining an "AV1 Image Item Data" term and using that in the new update. Take a look and let me know what you think...